### DISCUSSION TITLE
joint_angle

### DISCUSSION THREAD

**Discussion by caolikang** [roboterax/humanoid-gym#38] [Q&A] [answered]
Could you please let me know which information in the policy1.pt file contains the real-time joint angles of the robot's movement?

**Comment by zlw21gxy** [comment, marked as answer]
In a single observation, the `obs_buf[:, 5:17]` contains the joint position. Since we concatenate 15 frames, you can do the math. You can reference this https://github.com/roboterax/humanoid-gym/blob/620c5fac0cebbad60a36caa35b5ff30a0c78e4e2/humanoid/envs/custom/humanoid_env.py#L237 and https://github.com/roboterax/humanoid-gym/blob/620c5fac0cebbad60a36caa35b5ff30a0c78e4e2/humanoid/envs/custom/humanoid_env.py#L258.
